app/models/image.rb

```diff
@@ -0,0 +1,3 @@
+class Image < ApplicationRecord
+  validates :link, format: URI.regexp(%w[http https])
```

Comment:
Consider validating the "presence" (ie, that the attribute exists and is not blank) separately from the "format", so that we can get more meaningful/specific error messages ("can't be blank" vs "is invalid")